src/start/WelcomeScreen.js

```diff
@@ -0,0 +1,44 @@
+/* @flow */
+import React, { PureComponent } from 'react';
+import { View, StyleSheet } from 'react-native';
+
+import connectWithActions from '../connectWithActions';
+import type { Actions } from '../types';
+import { Screen, Centerer, ZulipButton } from '../common';
+
+const componentStyles = StyleSheet.create({
+  divider: {
+    height: 20,
+  },
+});
+
+type Props = {
+  actions: Actions,
+};
+
+class WelcomeScreen extends PureComponent<Props> {
+  props: Props;
+
+  render() {
+    const { actions } = this.props;
+    return (
+      <Screen title="Welcome!" centerContent padding>
+        <Centerer>
+          <ZulipButton
+            secondary
+            text="I have a Zulip account"
```

Comment:
Make use to add this strings in `i18n/translations/messages_en.json` too. 
We are using http://transifex.com/ for managing all translations.
Just add new new strings in `en` version and then with the help of `transifex` we will take care for other languages too.